src/components/deviceMap/deviceMap.js

```diff
@@ -168,7 +168,7 @@ class DeviceMap extends Component {
   render() {
     const { BingMapKey } = this.props;
     return (
-      <DashboardPanel title={lang.DEVICELOCATION} indicator={this.props.showSpinner} className="map-container">
+      <DashboardPanel title={lang.DEVICELOCATION} showSpinner={this.props.showInitialSpinner} indicator={this.props.showSpinner} className="map-container">
```

Comment:
I don't see any difference between showInitialSpinner and showSpinner. The reason for showing the spinner doesn't matter, only whether or not to show it.

Reply:
showSpinner is for the refresh, placed in the headerpanel and showInitialSpinner is for the on load and filter change, placed in the panelContents.

Reply:
Ok, these names need to be changed then to better indicate what they do. There is no obvious difference between `showSpinner` and `indicator`. They appear to be the same. Perhaps `showHeaderSpinner` and `showContentSpinner`.